components/logins/src/encryption.rs

```diff
@@ -236,6 +237,7 @@ impl NSSKeyManager {
     /// Initialize new `NSSKeyManager` with a given `PrimaryPasswordAuthenticator`.
     /// There must be a previous initializiation of NSS before initializing
     /// `NSSKeyManager`, otherwise this panics.
+    #[uniffi::constructor()]
```

Comment:
Does this compile with the keydb feature?  I think you might need to wrap the entire `impl` block with `#[uniffi::export]`.